src/battle/battle_command.c

```diff
@@ -71,6 +71,7 @@ BOOL BtlCmd_PokemonEncounter(BattleSystem *bsys, BattleContext *ctx) {
     case B_SIDE_PLAYER:
         break;
     case B_SIDE_OPPONENT:
+    {
```

Comment:
why do you keep adding random brackets into switch statements?

Reply:
if i don't, clangd reports an error